bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/container.jsx

```diff
@@ -32,13 +32,24 @@ export default withTracker((props) => {
 
   return {
     voiceUser: VoiceUsers.findOne({ intId: userId },
-      { fields: { muted: 1, listenOnly: 1, talking: 1 } }),
-    user: Users.findOne({ intId: userId },
       {
         fields: {
-          pin: 1, userId: 1, name: 1,
+          muted: 1, listenOnly: 1, talking: 1, joined: 1,
```

Comment:
What's the role of the `joined` field here? I don't see it being used.

Reply:
Marking as resolved - according to @frankemax this is actually a post-facto fix from something introduced in https://github.com/bigbluebutton/bigbluebutton/pull/14590. That prop is used in `user-status`, but was missing from the fetch.

Needs to be backported to 2.5

Reply:
Backported in https://github.com/bigbluebutton/bigbluebutton/pull/14965